src/components/AssetScanner/Webcam/Webcam.tsx

```diff
@@ -142,11 +142,7 @@ export class Webcam extends Component<WebcamProps, WebcamState> {
     }
     try {
       this.stream = stream;
-
-      if (this.video) {
-        this.video.srcObject = stream;
-        this.video.play();
-      }
+      this.video && ((this.video.srcObject = stream), this.video.play());
```

Comment:
😮  that looks like minified code, could you add a comment on why do you need it in this cryptic form?

Reply:
Done